card_class.py

```diff
@@ -0,0 +1,110 @@
+"""
+Module card2.
+Definit une classe Card et un petit cas-test qui demontre son utilite.
+TODO: 
+  - coder la fonction str
+  - le init qui prend 0 ou 2 parametres (indication : chercher sur StackOverflow "python optional arguments in initializer of python class")
+  - coder points ; d'ailleurs, est-ce une fonction ou un attribut ? 
+  - corriger le bug : qui est que le 10 est plus fort que Dame Roi Valet (7 8 9 V D R 10 As) 
+  - bonus : ajouter les atouts
+  - bonus : ajouter un setter qui empeche couleur = "VERTE"
+"""
+from enum import Enum
+
+COEUR, CARREAU, PIQUE, TREFLE = (0, 1, 2, 3)
+SEPT, HUIT, NEUF, DIX, VALET, DAME, ROI, AS = (7, 8, 9, 10, 11, 12, 13, 14)
+
+class Color(Enum):
+	COEUR = 0
+	CARREAU = 1
+	PIQUE = 2
+	TREFLE = 3
```

Comment:
Plus propre comme ça je trouve
```suggestion
Color = Enum("Color", "coeur carreau pique trefle")
```